.drone.yml

```diff
@@ -120,13 +123,49 @@ pipeline:
       event: deployment
       environment: qa
 
-  deploy-to-prod:
+  deploy-to-wcs-qa:
     image: quay.io/ukhomeofficedigital/kd:v1.16.0
     environment:
+      - KUBE_NAMESPACE=wcs-qa
+      - ENVIRONMENT=qa
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_docs_converter_wcs_qa
+    commands:
+      - source version.txt
+      - echo $VERSION
+      - echo $KUBE_NAMESPACE
+      - cd kube-hocs-docs-converter
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: qa
+
+  deploy-to-cs-prod:
+    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+    environment:
+      - KUBE_NAMESPACE=cs-prod
+      - ENVIRONMENT=prod
+      - KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_docs_converter_cs_prod
+    commands:
+      - echo $VERSION
+      - echo $KUBE_NAMESPACE
+      - cd kube-hocs-docs-converter
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: prod
```

Comment:
This will deploy at the same time as wcs-prod.

Reply:
True, but the changes for QA and PROD will be done in the next wave, as stated in the description of this PR.